LayoutTests/imported/w3c/web-platform-tests/css/css-lists/parsing/list-style-valid.html

```diff
@@ -12,12 +12,24 @@
 <body>
 <script>
 test_valid_value('list-style', 'none');
+test_valid_value('list-style', 'disc outside none', 'outside');
```

Comment:
Serialization of specified values is not much well defined, so producing `disc outside none` may not be that bad. What do other browser do?

Reply:
Chrome matches this patch, Firefox follows the spec and omits values when it does not change the meeting of the final value.

Reply:
The tests are already using shortest form, OK!